apibinding/examples/example_py_3_11.py

```
@@ -0,0 +1,87 @@
+        if rsp["state"] == "Done":
+            return json.loads(rsp["result"])
+        if time.time() >= deadline:
+            raise Exception("query API result timeout, job uuid: %s" % job_uuid)
+
+        time.sleep(1)
+        print("Job[uuid:%s] is still in processing" % job_uuid)
+
+
+
+def error_if_fail(rsp):
+    success = list(rsp.values())[0]["success"]
```

Ruff (0.15.10)

[warning] Prefer `next(iter(rsp.values()))` over single element slice

Replace with `next(iter(rsp.values()))`

(RUF015)